blog/_posts/2024-04-28-moving-extensionmanager-to-aks-control-plane

```diff
@@ -0,0 +1,57 @@
+---
+title: "Moving ExtensionManager to AKS Control Plane"
+description: "Learn how ExtensionManager in AKS control plane safeguards extension integrity, simplifies networking setup, and reduces identity footprint for your extensions experience on AKS."
+date: 2025-04-24 # date is important. future dates will not be published
+author: Jane Guo # must match the authors.yml in the _data folder
+categories: 
+- extensions 
+# - general
+# - operations
+# - networking
+# - security
+# - developer
+# - add-ons
+# - extensions
+---
+
+## What is ExtensionManager?
+
+If you have used any of the cluster extensions for AKS, such as Dapr, Flux (GitOps), Azure Machine Learning, Azure Backup, Azure Container Storage, and Azure App Configuration, or any 3rd party Kubernetes Apps on Azure Marketplace like Datadog AKS Cluster Extension, Isovalent Cilium Enterprise, and Cast AI agent, you may be familiar with ExtensionManager. ExtensionManager is a key component responsible for the cluster extension lifecycle operations (Create, Upgrade, Delete, Reconciliation). It periodically polls for version updates on extension instances and reconciles extension configuration settings, ensuring that extensions are managed effectively. 
+
+## What are cluster extensions?
+
+Cluster extensions are pre-packaged applications that provide an Azure Resource Manager-driven experience for installation and lifecycle management of different Azure capabilities on top of your Kubernetes cluster.
+* For more information about specific cluster extensions for AKS, see [Deploy and manage cluster extensions for Azure Kubernetes Service (AKS)](https://learn.microsoft.com/en-us/azure/aks/cluster-extensions?tabs=azure-cli).
```

Comment:
Should we link to the same article twice?

Reply:
You could combine the sentences.  For more information about available or specific cluster extensions...